client/client.go

```diff
@@ -0,0 +1,75 @@
+package client
+
+import (
+	"context"
+	"log"
+	"net"
+	"os"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+
+	pb "github.com/dapr/go-sdk/dapr/proto/runtime/v1"
+)
+
+const (
+	daprPortDefault    = "4000"
```

Comment:
change default port to 50001 (the default for kubernetes) for the NewClient() works in k8s

Reply:
done